hydrabus_framework/core/engine.py

```diff
@@ -0,0 +1,122 @@
+import inspect
+import os
+import pkgutil
+import sys
+from hydrabus_framework.core.logger import Logger
+from hydrabus_framework.core.command.run import run_module
+from hydrabus_framework.core.command.show import show
+from hydrabus_framework.core.command.set_options import set_options
+from hydrabus_framework.core.command.use import use
+from hydrabus_framework.core.command.back import back
+from hydrabus_framework.core.command.quit import hbf_exit
+from hydrabus_framework.core.command.help import hbf_help
+from hydrabus_framework.modules.base import BaseModule, ABaseModule
+from importlib import import_module
+from prompt_toolkit.shortcuts import PromptSession
+from prompt_toolkit.styles import Style
+
+
+class HydraFramework:
+    """
+    Framework core engine
+    """
+    def __init__(self):
+        self.logger = Logger()
+        self.app_path = sys.path[0]
+        self.current_module = BaseModule()
+        self.modules = self._list_modules()
+        self.prompt_style = Style.from_dict({
+            # User input (default text).
+            '': '#ffffff',
+
+            # Prompt.
+            'username': '#3399ff',
+            'pound': '#3399ff',
+            'path': '#ff0000 bold',
+        })
+        self.prompt = [
+            ('class:username', '[hbf] '),
+            ('class:path', ''),
+            ('class:pound', '> '),
+        ]
+        self.command = [
```

Comment:
Should be named commands. It is missleading in the rest of the code

Reply:
Done [ #cf0c342](https://github.com/hydrabus-framework/framework/commit/cf0c342ce9ebce84e27ee20c6e1a789a903ab5e3)